apps/notifications/src/alert/controllers/chain-events/chain-events.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { MsgCloseDeployment } from '@akashnetwork/akash-api/v1beta3';
+import { Injectable } from '@nestjs/common';
+
+import { ChainBlockCreatedDto } from '@src/alert/dto/chain-block-created.dto';
+import { MsgCloseDeploymentDto } from '@src/alert/dto/msg-close-deployment.dto';
+import { DeploymentBalanceAlertsService } from '@src/alert/services/deployment-balance-alerts/deployment-balance-alerts.service';
+import { RawAlertsService } from '@src/alert/services/raw-alerts/raw-alerts.service';
+import { PgBossHandler } from '@src/broker';
+
+@Injectable()
+export class ChainEventsController {
+  constructor(
+    private readonly rawAlertsService: RawAlertsService,
+    private readonly deploymentBalanceAlertsService: DeploymentBalanceAlertsService,
+  ) {}
+
+  @PgBossHandler({
+    key: 'blockchain.v1.block.created',
+    dto: ChainBlockCreatedDto,
+  })
+  async processBlock(block: ChainBlockCreatedDto) {
+    await this.deploymentBalanceAlertsService.alertFor(block);
+  }
+
+  @PgBossHandler({
+    key: MsgCloseDeployment['$type'],
+    dto: MsgCloseDeploymentDto,
+  })
+  async processDeploymentClosed(event: MsgCloseDeploymentDto) {
+    await this.rawAlertsService.alertFor(event);
```

Comment:
Ok we won't support all the message types generically like blockspy does, but with the rawAlertsService, we could implement something similar if needed. 👍

Reply:
this actually does support any message. the only difference is it's not subscribed to all of them by default. It's better add stuff when it's needed.